test/parallel/test-http2-client-promisify-connect.js

```diff
@@ -27,6 +28,21 @@ server.listen(0, common.mustCall(() => {
         assert.strictEqual(data, 'ok');
         client.close();
         server.close();
+        testConnectionError();
       }));
     }));
 }));
+
+// Error connection test
+function testConnectionError() {
+  const connect = util.promisify(http2.connect);
+
+  // Attempt to connect to an invalid port
+  connect('http://localhost:9999')
```

Comment:
```suggestion
  connect('http://localhost:9999')
```
There could be something running on this port, maybe a safer approach would be starting the server and refuse the connection

Reply:
Thanks, the change is done